packages/next/src/client/image-component.tsx

```diff
@@ -320,7 +320,7 @@ function ImagePreload({
     ...getDynamicProps(imgAttributes.fetchPriority),
   }
 
-  if (isAppRouter) {
+  if (isAppRouter && preload) {
```

Comment:
Looks good. We might even want to drop the `isAppRouter` check completely but we can start with this.